### ISSUE TITLE
Add task exec agent support

### ISSUE THREAD

**Issue by nohaapav** [swarmpit/swarmpit#93]

**Comment by nohaapav** [comment]
Depends on https://github.com/swarmpit/swarmpit/issues/225

**Comment by nohaapav** [comment]
Moving to 1.8 .. Logs have bigger priority in 1.7

**Comment by neygomon** [comment]
Bump...
When is it planned to implement this functionality ?!
Without it, it is impossible to use this product normally!

**Comment by nohaapav** [comment]
@neygomon If you need to exec into container you're doing something wrong in the first place. This has nothing to do with product.

Probably in 2.0 release, we have no roadmap ready yet.

**Comment by hlavki** [comment]
What's wrong with e.g. listing of kafka topics.

**Comment by nohaapav** [comment]
yes i'm saying that `docker exec` is wrong from security perspective and should be avoided .. Same applies for network attaching into existing container etc .. Even docker is using access authorization plugin to restrict access to given API's in ee. There is bunch of related blogs out there why you should avoid such practise, just google it.

I'm not saying it can't be helpful, but if you're going to write solid system with security in first place you shouldn't do so.

**Comment by hlavki** [comment]
You are right about security, but no one says that this feature is for production ready system.
Software development is process, where tools like this can be time saver. Especially in development phase on dev or test environment.

**Comment by nohaapav** [comment]
@hlavki It was just reaction to stupid comment `Without it, it is impossible to use this product normally` :) If one can't use swarmpit without exec that something is not right. I barely use that feature and for past 4 years containers are my bread and butter. 

We'll do our best to implement such feature as soon as possible, but there is a bunch of features with higher priority and not so much capacity left. But fingers crossed for 2.0 ;)
